### ISSUE TITLE
Explorer return a 500 error code when looking for a non existing 3bot record

### ISSUE THREAD

**Issue by zaibon** [threefoldfoundation/tfchain#330]
when searching for a 3bot record that doesn't exist, the explorer API gives a 500 error code while I think it would be more suited to return a 404.

```
▶ curl -i https://explorer.testnet.threefoldtoken.com/explorer/3bot/4
HTTP/2 500 
access-control-allow-methods: *
access-control-allow-origin: *
content-type: application/json; charset=utf-8
date: Fri, 15 Mar 2019 09:41:31 GMT
server: Caddy
content-length: 29

{"message":"3bot not found"}
```

**Comment by LeeSmet** [comment]
Should already be fixed in 85c5ec3012fadf9547933f240e8b397095821d1d, which should be live in the next deploy

**Comment by zaibon** [comment]
@LeeSmet ok perfect, thx
